db/migrations/20230327063144_users.sql

```diff
@@ -0,0 +1,14 @@
+-- migrate:up
+CREATE TABLE users (
+  id INT NOT NULL AUTO_INCREMENT PRIMARY KEY,
+  account_email VARCHAR(30) NOT NULL,
+  nickname VARCHAR(30) NOT NULL,
+  age_range VARCHAR(30) NOT NULL,
+  social_id INT NOT NULL,
```

Comment:
```suggestion
  social_id VARCHAR(30) NOT NULL,
```
실제 user info를 받아올 때 id 값 같은 경우에는 카카오 혹은 구글을 예로 들면, `123456789010`과 같은 큰 수가 들어오기 때문에 INT로 정의된 field에는 저장 되지 않는 경우가 있습니다. BIGINT 혹은 VARCHAR로 수정해주세요!